config/search/config.json

```diff
@@ -0,0 +1,182 @@
+{
+  "automatic_index_updates": true,
+  "persist_indexes": true,
+  "number_of_threads": 8,
+  "eager_indexing_groups": [
+    [
+      {
+        "name": "public",
+        "variables": []
+      }
+    ],
+    [
+      {
+        "name": "clean",
+        "variables": []
+      }
+    ]
+  ],
+  "default_settings": {
+    "analysis": {
+      "normalizer": {
+        "custom_sort_normalizer": {
+          "type": "custom",
+          "char_filter": [],
+          "filter": [
+            "lowercase",
+            "trim",
+            "asciifolding"
+          ]
+        }
+      }
+    }
+  },
+  "types": [
+    {
+      "type": "agenda-item",
+      "on_path": "agenda-items",
+      "rdf_type": [
+        "http://data.vlaanderen.be/ns/besluit#Agendapunt"
+      ],
+      "properties": {
+        "location_id": [
+          "^http://data.vlaanderen.be/ns/besluit#behandelt",
+          "http://data.vlaanderen.be/ns/besluit#isGehoudenDoor",
+          "http://data.vlaanderen.be/ns/mandaat#isTijdspecialisatieVan",
+          "http://data.vlaanderen.be/ns/besluit#bestuurt",
+          "http://data.vlaanderen.be/ns/besluit#werkingsgebied",
+          "http://mu.semte.ch/vocabularies/core/uuid"
+        ],
+        "time_specialization_location_name": [
+          "^http://data.vlaanderen.be/ns/besluit#behandelt",
+          "http://data.vlaanderen.be/ns/besluit#isGehoudenDoor",
+          "http://data.vlaanderen.be/ns/mandaat#isTijdspecialisatieVan",
+          "http://data.vlaanderen.be/ns/besluit#bestuurt",
+          "http://data.vlaanderen.be/ns/besluit#werkingsgebied",
+          "http://www.w3.org/2000/01/rdf-schema#label"
+        ],
+        "governing_body_location_name": [
+          "^http://data.vlaanderen.be/ns/besluit#behandelt",
+          "http://data.vlaanderen.be/ns/besluit#isGehoudenDoor",
+          "http://data.vlaanderen.be/ns/besluit#bestuurt",
+          "http://data.vlaanderen.be/ns/besluit#werkingsgebied",
+          "http://www.w3.org/2000/01/rdf-schema#label"
+        ],
+        "time_specialization_name": [
```

Comment:
**issue:** I understand why you prefix this property with `time_specialization`. But it's confusing, because the content is not the time specified governing-body name but the abstract governing-body name. 

agenda-item <- session -> governing-body with time specification -> abstract governing-body -> administrative-unit -> location  

What do you think of using `abstract_governing_body_name` instead?
```suggestion
        "abstract_governing_body_name": [
```

Reply:
I think it's a good idea!